src/components/Checkbox/Checkbox.tsx

```diff
@@ -41,16 +40,38 @@ const AminoCheckbox = styled.div<{ checked: boolean }>`
   }
 `;
 
+const StyledSubtitle = styled(Text)`
+  font-style: normal;
+`;
+
+const StyledLabel = styled(Text)``;
```

Comment:
This one I just use to refer to style child label from parent component instead of passing parameter down to children label tag. Should I think of a different way of doing that?
![image](https://user-images.githubusercontent.com/17950626/158908443-5d9fc3a8-5b65-47bd-869c-5eaf8ba8e508.png)

Reply:
Really cool! I think that is great 💯